README.md

```diff
@@ -1,5 +1,14 @@
 ## GraphStorm
-The GraphStorm is a graph machine learning (GML) framework for enterprise use cases. It simplifies the development, training and deployment of GML models for industry-scale graphs. It provides scalable training and inference pipelines for various GML models and is optimized for extremely large graphs (measured in billons of nodes and edges). To help develop SOTA models, the framework allows very large degree of customization by providing a large collection of configurations for customizing model implementations and training pipelines as well as modularization in the training pipeline that allows users to replace every component in the pipeline.
+GraphStorm is a graph machine learning (GML) framework for enterprise use cases.
+It simplifies the development, training and deployment of GML models for industry-scale graphs.
+It provides scalable training and inference pipelines for various Graph Machine Learning (GML) models
+and is optimized for extremely large graphs (measured in billons of nodes and edges).
+GraphStorm provides a collection of built-in GML models and users can train a GML model
```

Comment:
Can add a line break above here to break up into two paragraphs?

Reply:
the topic sentence of the paragraph is "GraphStorm is a graph machine learning (GML) framework for enterprise use cases."
this paragraph isn't very long. breaking it into two paragraphs will make each paragraph very short.